### ISSUE TITLE
Failing to Load PyTest in Test Explorer

### ISSUE THREAD

**Issue by GregHattJr** [kondratyev-nv/vscode-python-test-adapter#196]
So the tests appear properly under the Python section. I can run them and see their results and what-not. However, no tests are being shown in the top left Test Explorer section. The only output from the logs for the Test Adapter are:

```
2020-10-09T04:01:54.572Z INFO unittest at 'ui-resources-django': Reading configuration for workspace ui-resources-django
2020-10-09T04:01:54.573Z  DBG unittest at 'ui-resources-django': usingNewInterpreterStorage feature flag is 'true'
2020-10-09T04:01:54.573Z INFO pytest at 'ui-resources-django': Reading configuration for workspace ui-resources-django
2020-10-09T04:01:54.573Z  DBG pytest at 'ui-resources-django': usingNewInterpreterStorage feature flag is 'true'
2020-10-09T04:01:58.021Z CRIT unittest at 'ui-resources-django': Test loading failed: TypeError: extension.exports.settings.getExecutionDetails is not a function
2020-10-09T04:01:58.022Z CRIT pytest at 'ui-resources-django': Test loading failed: TypeError: extension.exports.settings.getExecutionDetails is not a function
```

I've read through the docs and other issues and am at a lose on how to proceed.

**Comment by kondratyev-nv** [comment]
@GregHattJr Thank you for the issue! It looks like a bug introduced in the latest version. I'll try to fix it and release a new version this weekend. Meanwhile, I can suggest to rollback to a previous version. To do so, click on the Manage button of the extension in the Extensions panel and Select "Install Another Version...".
<img width="289" alt="image" src="https://user-images.githubusercontent.com/4085884/95571617-a21d6a00-0a20-11eb-8b6d-c0858d37e081.png">

**Comment by kondratyev-nv** [comment]
@GregHattJr I've just published a new version 0.6.1 that should not break test discovery and execution when there are any errors getting pythonPath from the Python Extension. Please, update and let me know if you still have issues discovering your tests. Unfortunately, I was not able to reproduce the error, so I've extended logging when auto-detection breaks. Please, paste any errors here, so I will be able to dig into it.

**Comment by kondratyev-nv** [comment]
@GregHattJr I'm closing this issue assuming this issue is resolved. Please, reopen it or comment if you have any problems with that. In case you have any other problems, please, feel free to create a new issue.
